staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -163,6 +165,9 @@ func (s *SecureServingOptions) AddFlags(fs *pflag.FlagSet) {
 	}
 	fs.IntVar(&s.BindPort, "secure-port", s.BindPort, desc)
 
+	fs.BoolVar(&s.DisableHTTP2, "disable-http2", s.DisableHTTP2,
+		"If true, HTTP2 will be disabled [default=false]")
```

Comment:
I think that Jordan's comment still applies here: https://github.com/kubernetes/kubernetes/pull/116762/files#r1261602446

Maybe you could go with something like `disable-serving-http2` and try to improve the help message to make it clearer to the users?

Reply:
Updated please take a look

Reply:
Looks good to me now